examples/bayesian_linear_regression_test.py

```diff
@@ -63,22 +63,22 @@ def predict(self, xs, zs):
         y_pred = tf.reduce_mean(tf.matmul(x_test, W) + b, 1)
         return y_pred
 
-def build_toy_dataset(n_data=40, coeff=np.random.randn(10), noise_std=0.1):
+def build_toy_dataset(n_minibatch=40, coeff=np.random.randn(10), noise_std=0.1):
```

Comment:
This one refers to the number of data points for building the toy data set. So maybe something like `N`?